homework/04LessonFour.js

```diff
@@ -77,7 +82,7 @@ var nestedObjects = {
   }
 };
 // update the starring property to have the value 'Tim Robbins'
-
+nestedObjects.users['bob roberts'] = {'starring':'Tim Robbins'};
```

Comment:
You need to assign just the value of "Tim Robbins" to this problem, not the key value pair.

Bracket and dot notation is all that is required to do so.